example-config.yml

```diff
@@ -365,6 +369,37 @@ itinerary:
 #         modes: "CAR_HAIL, CAR_RENT"
 #       480:
 #         msg: No available transit routes or rideshare/carshare service at origin.
+  # fr-FR:
+  #   name: French (France)
+  #   flag: 🇫🇷
```

Comment:
We're not using `flag` anymore are we?

Reply:
Good catch, changed in https://github.com/opentripplanner/otp-react-redux/pull/525/commits/a6de95114aed8b19ec7c8910f2e5e500a88a524d